tests/src/test/java/io/streamnative/pulsar/handlers/mqtt/ProxyTLSTest.java

```diff
@@ -46,8 +52,8 @@ protected MQTTServerConfiguration initConfig() throws Exception {
         return mqtt;
     }
 
-    @Test(dataProvider = "mqttTopicNames", timeOut = TIMEOUT)
-    public void testConnectionUsingTLS(String topicName) throws Exception {
+    @Test(dataProvider = "mqttTopicNames", timeOut = TIMEOUT, priority = 3)
```

Comment:
Does the test execution order affect the test result?

Reply:
Yes, subscribe wildcard topics should be the first one. Because other test case methods use the same topics that may be lead the newly added wildcard test method to receive more message.